bancor_research/bancor_emulator/v3/spec/network.py

```diff
@@ -145,8 +145,10 @@ def begin_cooldown(
         tkn_name: str,
         user_name: str,
         timestamp: int = 0,
+        by_ptkn_amt: bool = False,
```

Comment:
suggest renaming to `is_unit_tkn` (or similar)

Reply:
I've split the cooldown function into two functions, so this flag's gone now